Expected error message should hide in logback-test.xml

There is some error message which is expected behaviour, it is better to hide them during config in logback-test.xml

```
Running org.apache.shardingsphere.elasticjob.lite.integrate.disable.ScheduleDisabledJobIntegrateTest
[ERROR] 2020-07-13 00:28:14,497 --438323506050282_test_job_Worker-1-- [org.apache.shardingsphere.elasticjob.infra.handler.error.impl.LogJobErrorHandler] Job '438323506050282_test_job' exception occur in job processing 
org.apache.shardingsphere.elasticjob.infra.exception.JobSystemException: Job timeout. timeout mills is 100.
	at org.apache.shardingsphere.elasticjob.lite.api.listener.AbstractDistributeOnceElasticJobListener.handleTimeout(AbstractDistributeOnceElasticJobListener.java:105)
	at org.apache.shardingsphere.elasticjob.lite.api.listener.AbstractDistributeOnceElasticJobListener.beforeJobExecuted(AbstractDistributeOnceElasticJobListener.java:74)
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJobFacade.beforeJobExecuted(LiteJobFacade.java:145)
	at org.apache.shardingsphere.elasticjob.executor.ElasticJobExecutor.execute(ElasticJobExecutor.java:98)
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJob.execute(LiteJob.java:52)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
[ERROR] 2020-07-13 00:28:14,607 --438323506050282_test_job_Worker-1-- [org.apache.shardingsphere.elasticjob.infra.handler.error.impl.LogJobErrorHandler] Job '438323506050282_test_job' exception occur in job processing 
org.apache.shardingsphere.elasticjob.infra.exception.JobSystemException: Job timeout. timeout mills is 100.
	at org.apache.shardingsphere.elasticjob.lite.api.listener.AbstractDistributeOnceElasticJobListener.handleTimeout(AbstractDistributeOnceElasticJobListener.java:105)
	at org.apache.shardingsphere.elasticjob.lite.api.listener.AbstractDistributeOnceElasticJobListener.afterJobExecuted(AbstractDistributeOnceElasticJobListener.java:100)
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJobFacade.afterJobExecuted(LiteJobFacade.java:152)
	at org.apache.shardingsphere.elasticjob.executor.ElasticJobExecutor.execute(ElasticJobExecutor.java:111)
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJob.execute(LiteJob.java:52)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
```

--- a/elasticjob-lite/elasticjob-lite-core/src/test/resources/logback-test.xml
+++ b/elasticjob-lite/elasticjob-lite-core/src/test/resources/logback-test.xml
@@ -37,4 +37,6 @@
     </root>
     
     <logger name="org.apache.shardingsphere.elasticjob.lite.internal.snapshot.SnapshotService" level="OFF" />
+    <logger name="org.apache.shardingsphere.elasticjob.infra.handler.error.impl.LogJobErrorHandler" level="OFF" />
+    <logger name="org.apache.curator.framework.listen.MappingListenerManager" level="OFF" />
 </configuration>

